storage/tests/unit/test_bucket.py

```diff
@@ -18,6 +18,15 @@
 import mock
 
 
+def _make_connection(*responses):
+    import google.cloud.storage._http
+
+    mock_connection = mock.create_autospec(google.cloud.storage._http.Connection)
+    mock_connection.user_agent = "testing 1.2.3"
+    mock_connection.api_request.side_effect = list(responses)
+    return mock_connection
+
```

Comment:
With this PR diffs we are no more able to use `_Client()` class in some tests. I've rewrote them with connection mocks and `Client` objects